src/components/NewCommentForm.tsx

```diff
@@ -13,25 +76,36 @@ export const NewCommentForm: React.FC = () => {
             type="text"
             name="name"
             id="comment-author-name"
+            value={name}
             placeholder="Name Surname"
-            className="input is-danger"
+            className={classNames('input', {
+              'is-danger': !isNameAllright,
+            })}
+            onChange={(e) => {
+              setName(e.target.value);
+              setIsNameAllright(true);
+            }}
```

Comment:
Create separate handler instead of anonymous function. Try fix all places

Reply:
done